<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>751</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{background-color:qlineargradient(spread:reflect, x1:0.631, y1:0.840909, x2:0.266591, y2:0.13, stop:0 rgba(232, 255, 255, 255), stop:1 rgba(230, 147, 230, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>MIR</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ingresa o crea una nueva cuenta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_ingresar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_crear_cuenta">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string>Crear nueva cuenta</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
